### ISSUE TITLE
Kinght Move has bugs

### ISSUE THREAD

**Issue by ShowmickKar** [ShowmickKar/Chess-Game#2]

**Comment by Dragjon** [comment]
Yes, the files for the knights and kings are messed up as both of them start with the letter k, that is why we should use the letter 'n' for knights just like in chess.

**Comment by Dragjon** [comment]
Current bugfixes version can be found here https://github.com/Dragjon/Chess-Game-Bugfix

**Comment by Dragjon** [comment]
You can update your github repo

**Comment by ShowmickKar** [comment]
Hi @Dragjon, Sorry I just saw your pull request. I will review the changes and merge it to my repository when I get the time. Thank you for your contribution!
